store/sqlstore/channel_store.go

```diff
@@ -1812,6 +1812,69 @@ func (s SqlChannelStore) UpdateLastViewedAt(channelIds []string, userId string)
 	return times, nil
 }
 
+// CountUnreadSince gives the number of posts that a channel has since some date, this doesn't follow any threads.
+// this allows us to know how many posts have bean read already.
+func (s SqlChannelStore) CountUnreadSince(channelID string, since int64) (int64, *model.AppError) {
+	countUnreadQuery := `
+	SELECT count(*)
+	FROM Posts
+	WHERE
+		ChannelId = :channelID
```

Comment:
A couple minor nits, but we usually capitalize named parameters (so this line would be `ChannelId = :ChannelId`) and we use single quotes in SQL statements